config/chainbridge-runtime-config.cere.dev.js

```diff
@@ -52,16 +52,15 @@ window.__RUNTIME_CONFIG__ = {
             decimals: cereTokenDecimals,
           },
         ],
-        destinationChain: true,
       },
       {
-        chainId: 5,
+        chainId: 0,
```

Comment:
Why did you change?

Reply:
In our project, we have configured the 0 as the id for Ethereum Blockchain.